### PR TITLE
[scrot] Improve saving of screenshot and extension name

### PR DESCRIPTION
**Pull request by benedictdudel**
Make use of the environment variable `XDG_PICTURES_DIR` when creating
screenshot. If those variable is not set we save the screenshot in the
temp directory.

Additionally we need to rename the extension name to give users a hint
which don't know `scrot`.

@ManuelSchneid3r Do you know a way to delay the execution of the command? `scrot` doesn't support a value below 1 second.

@schoettl Thanks for your review. I tried to implement you suggestions in the old pull request. Would like to see your feedback :)

### CODE REVIEW

File: Scrot.py [hunk 1]
```diff
@@ -25,7 +27,10 @@
     raise Exception("'xclip' is not in $PATH.")
 
 iconPath = iconLookup("camera-photo")
-fileName = "%Y-%m-%d-%T-screenshot.png"
+screenshotDirectory = os.getenv("XDG_PICTURES_DIR", tempfile.gettempdir())
```
Comment (schoettl) [inline Scrot.py]: In my understanding, the XDG dirs are a bit more complicated to query: The environment variables are normally not defined. They would only be defined if you sourced the `~/.config/user-dirs.dirs` in a shell.

To query the directory correctly, it looks like you have to call this command: `xdg-user-dir PICTURES` - its output is the path to use. If empty, use the temp dir. See the last lines of this link for reference: https://wiki.archlinux.org/index.php/XDG_user_directories

Then the xdg package (or xdg-user-dir?) should be added as dependency.

File: Scrot.py [hunk 2]
```diff
@@ -25,7 +27,10 @@
...
+fileName = os.path.join(screenshotDirectory, "%Y-%m-%d-%T-screenshot.png")
+xclipCommand = "xclip -selection c -t image/png < $f"
+scrotCommand = ["scrot", "--delay", "1", "--exec", xclipCommand, fileName]
```
Comment (schoettl) [inline Scrot.py]: Looks very clean now :)

**Comment by ManuelSchneid3r** [comment]
@benedictdudel use a FuncActio

File: Scrot.py (hunk 1 above)
Comment (benedictdudel) [reply Scrot.py]: ah, thanks I totally missed that..

**Comment by ManuelSchneid3r** [comment]
@benedictdudel Sorry wrote from my phone and hit enter by accident GihubMobile does not allow to edit messages. Use a FuncAction and sleep before run.

**Comment by benedictdudel** [comment]
@ManuelSchneid3r No problem :) just needed a hint :P

**Pull request merged**